deploy/helm/elastic-agent/templates/agent/cluster-role.yaml

```diff
@@ -11,9 +12,9 @@ metadata:
     {{- with ($presetVal).labels -}}
     {{ toYaml . | nindent 4 }}
     {{- end }}
-  {{- with ($presetVal).annotations }}
+  {{- with ($presetVal).clusterRole.annotations -}}
```

Comment:
Shouldn't this be both the preset-wide annotations *and* the ClusterRole-specific ones?

Reply:
I debated in my head about it, and I chose to not "merge" the two as I deemed it cleaner to have separate annotations between {deployment|statefulset|daemonset}, clusterrole, and serviceaccount. That said, no strong feelings, hence, sure I will add also the preset annotations 🙂

Reply:
Hm, maybe this isn't that useful? In hindsight, I was thinking of labels, where having a common set for all resources makes a lot more sense than annotations, which tend to be tool-specific. If you don't want to merge them, I'm ok with it.

Reply:
oops missed this one and I merged them 🙂 have a look and if you don't like it I will revert it